File: 백준/Bronze/14659. 한조서열정리하고옴ㅋㅋ/한조서열정리하고옴ㅋㅋ.c
#include <stdio.h>
#include <stdlib.h>
int main(void) {
    int n, tar = 0, cnt = 0, maxn = 0, *arr;
    scanf("%d", &n);
    arr = (int*)malloc(sizeof(int) * n);
    for (int i = 0; i < n; i++)
        scanf("%d", &arr[i]);
    for (int i = 0; i < n; i++) {
        if (tar < arr[i]) {
            tar = arr[i];
            cnt = 0;
        } else
            cnt += 1;
        if (maxn < cnt)
            maxn = cnt;
    }
    printf("%d", maxn);
    return 0;
}

File: 백준/Bronze/14659. 한조서열정리하고옴ㅋㅋ/한조서열정리하고옴ㅋㅋ.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n=int(input())
arr=[*map(int,input().split())]
tar,cnt,maxn=0,0,0
for i in arr:
    if tar<i:
        tar=i
        cnt=0
    else:
        cnt+=1
    if maxn<cnt:
        maxn=cnt
print(maxn)